langchain/src/retrievers/parent_document.ts

```diff
@@ -45,6 +55,14 @@ export class ParentDocumentRetriever extends MultiVectorRetriever {
 
   protected parentK?: number;
 
+  protected retriever: ScoreThresholdRetriever<VectorStore> | undefined;
```

Comment:
Should we instead allow an arbitrary retriever to be passed here that overrides the default? Could imagine MMR etc. being useful

Reply:
Yeah agreed

Should probably update how the options for the retriever are passed in too

Working on it now

Reply:
Updated code & docs